src/views/ProjectProfile.vue

```diff
@@ -120,6 +120,12 @@ export default {
   text-align: center;
 }
 
+#narrative {
+    a {
+        color: blue;
+    }
+}
```

Comment:
I don't think we need this. Now that we aren't making all links white in `App.vue`, blue links are the default unless we specify the `color-change` class, right?